lib/Yam/Agama/agama_base.pm

```diff
@@ -26,14 +29,15 @@ sub upload_traces {
 }
 
 sub upload_agama_logs {
-    select_console 'root-console';
     save_and_upload_log('agama logs store', "/tmp/agama-logs.tar.gz");
+}
+
+sub upload_system_logs {
     save_and_upload_log('journalctl -b > /tmp/journal.log', "/tmp/journal.log");
-    upload_traces();
 }
 
-sub post_run_hook {
-    upload_agama_logs();
+sub upload_browser_automation_dumps {
```

Comment:
please, instead of create a wrapper function, just rename `upload_traces` to this new name.

Reply:
Updated.